examples/verify_tsqrtP_calibration.py

```diff
@@ -0,0 +1,150 @@
+
+V_atten_ratio = 102.35  # attenutation ratio
+skip_plots = 33  # diagnostic -- set this to None, and there will be no plots
+linear_threshold = 100e-6
+slicewidth = 2e6
+
+
+with psd.figlist_var() as fl:
+    for filename, nodename in [
+        (
+            "240816_amp0p1_fin_pulse_calib.h5",
+            "pulse_calib_1",
+        ),  # low power
+        #(
+        #    "240805_test_calib_amp0p1_a_pulse_calib.h5",
+        #    "pulse_calib_2",
+        #),  # low power
+        #(
+        #    "240805_test_calib_amp0p2_a_pulse_calib.h5",
+        #    "pulse_calib_1",
+        #),  # low power
+        #(
+        #    "240805_test_calib_amp1_pulse_calib.h5",
+        #    "pulse_calib_1",
+        #),  # low power
+    ]:
+        d = psd.find_file(
+            filename, expno=nodename, exp_type="ODNP_NMR_comp/test_equipment"
+        )
+        assert (
+            d.get_prop("postproc_type") is not None
+        ), "No postproc type was set upon acquisition"
+        amplitude = d.get_prop("acq_params")["amplitude"]
+        fl.basename = f"amplitude = {amplitude}"
+        if not d.get_units("t") == "s":
+            print(
+                "units weren't set for the t axis or else I can't read them from the hdf5 file!"
+            )
+            d.set_units("t", "s")
+        d *= V_atten_ratio
+        d /= np.sqrt(50)  # V/sqrt(R) = sqrt(P)
+        pulse_lengths = d.get_prop("programmed_t_pulse_us")
+
+        def switch_to_plot(d, j):
+            thislen = pulse_lengths[j]
+            fl.next(f"pulse length = {thislen}")
+
+        def indiv_plots(d, thislabel, thiscolor):
+            if skip_plots is None:
+                return
+            for j in range(len(d["beta"])):
+                if j % skip_plots == 0:
+                    switch_to_plot(d, j)
+                    fl.plot(
+                        d["beta", j],
+                        alpha=0.2,
+                        color=thiscolor,
+                        label=thislabel,
+                    )
+
+        indiv_plots(d, "raw", "blue")
+        # {{{ data is already analytic, and downsampled to below 24 MHz
+        indiv_plots(abs(d), "analytic", "orange")
+        d.ft("t")
+        # {{{ apply frequency filter
+        d.ift("t")
+        SW = 1 / (d["t"][1] - d["t"][0])  # sample rate
+        carrier = (
+            d.get_prop("acq_params")["carrierFreq_MHz"] * 1e6
+        )  # signal frequency
+        fn = SW / 2  # nyquist frequency
+        n = np.floor(
+            (carrier + fn) / SW
+        )  # how far is the carrier from the left side of the spectrum (which is at SW/2), in integral multiples of SW
+        nu_a = (
+            carrier - n * SW
+        )  # find the aliased peak -- again, measuring from the left side
+        center = SW - abs(nu_a)
+        d.ft("t")
+        d["t" : (0, center - 0.5 * slicewidth)] *= 0
+        d["t" : (center + 0.5 * slicewidth, None)] *= 0
```

Comment:
as with the other PR, don't do this center thing.  If the signal aliases to negative, then it aliases to negative.

Reply:
we discussed that we had thrown out negative frequencies and so this will not work